File: meeting/template/meeting.js
process.stdin.resume();
process.stdin.setEncoding("utf-8");

let inputString = "";
let currentLine = 0;

process.stdin.on("data", (inputStdin) => {
    inputString += inputStdin;
});

process.stdin.on("end", () => {
    inputString = inputString.split("\n");
    main();
});

function readLine() {
    return inputString[currentLine++];
}

/**
 * @param {number[][]} schedules - An array of existing meeting schedules, where each schedule is represented as [start, end] pair.
 * @param {number[]} urgent_meeting - The urgent meeting to be inserted, represented as [start, end] pair.
 * @returns {number[][]} The updated schedule with merged intervals, where each element is an [start, end] pair representing a meeting.
 */
function reschedule(schedules, urgent_meeting) {

}

function main() {
    const schedules = [];
    const n = parseInt(readLine());
    for (let i = 0; i < n; i++) {
        const [start, end] = readLine().split(" ").map(Number);
        schedules.push([start, end]);
    }

    const [urgentStart, urgentEnd] = readLine().split(" ").map(Number);
    const urgentMeeting = [urgentStart, urgentEnd];

    const result = reschedule(schedules, urgentMeeting);
    for (const schedule of result) {
        console.log(`${schedule[0]} ${schedule[1]}`);
    }
}

File: meeting/template/meeting.cpp
#include <bits/stdc++.h>
using namespace std;

vector<vector<int>> reschedule(vector<vector<int>> &schedules,
                               vector<int> &urgent_meeting) {

}

int main() {
  int n, start, end;
  vector<vector<int>> schedules;
  vector<int> urgent_meeting(2);

  cin >> n;
  while (n--) {
    cin >> start >> end;
    schedules.push_back({start, end});
  }
  cin >> urgent_meeting[0] >> urgent_meeting[1];

  vector<vector<int>> result = reschedule(schedules, urgent_meeting);
  for (vector<int> schedule : result) {
    cout << schedule[0] << " " << schedule[1] << endl;
  }
}
